test/fixtures/rules/std_rule_cases/L036.yml

```diff
@@ -37,7 +37,7 @@ test_multiple_select_targets_all_on_the_same_line:
   fail_str: |
     select a, b, c
     from x
-  fix_str: "select\na, b, c\nfrom x\n"
+  fix_str: "select\na,\nb,\nc\nfrom x\n"
```

Comment:
Would you mind multiline yamling this just to make it a bit clearer to the eye?

Reply:
Done!